frontend/ui/impl/build.gradle

```
@@ -0,0 +1,23 @@
+lombok()
+slf4jLambdaApi()
+vavr()
+
+dependencies {
+  implementation project(':binding')
+  implementation project(':branchLayout:api')
+  implementation project(':backend:api')
+  implementation project(':frontend:compatibilityBridge')
+  implementation project(':frontend:defs')
+  implementation project(':frontend:dataKeys')
+  implementation project(':frontendGraph:api')
+  implementation project(':frontend:vfsUtils')
+  implementation project(':frontendFile')
+  implementation project(':frontend:providerServices')
+  implementation project(':frontend:resourceBundles')
+
+  api project(':frontend:ui:api')
```

Huh I see it was in the original `frontendUiImpl/build.gradle`... but I'm not sure why that was needed in the first place 🤔 

There's in principle no need for `...:impl` modules to expose of its dependencies as `api`, since `...:impl` are NOT being depended upon themselves... pls change this one to:

```suggestion
  implementation project(':frontend:ui:api')
```

and my hypothesis is that the project will still build.

Extra reading: https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation, without the parts abour publishing... coz we don't publish anything to Maven/Ivy, only to the (somewhat simplistic in comparison) Jetbrains Marketplace.